apps/files_external/lib/Service/StoragesService.php

```diff
@@ -424,6 +426,10 @@ public function updateStorage(StorageConfig $updatedStorage) {
 			}
 		}
 
+		if ($this->dbConfig->hasHomeFolderOverwriteMount()) {
+			$this->appConfig->setValueBool('files', 'homeFolderOverwritten', true);
+		}
```

Comment:
When is it ever set to false?
Shouldn’t it be:
```suggestion
		$this->appConfig->setValueBool('files', 'homeFolderOverwritten', $this->dbConfig->hasHomeFolderOverwriteMount()));
```

Reply:
I think it is never set to `false`, it is either `true` when needed or undefined.

Reply:
In the end, it indeed makes sense to set it to `false` if needed during an external storage update.